radio/src/targets/horus/CMakeLists.txt

```diff
@@ -294,7 +294,7 @@ MACRO(GET_GCC_INCLUDE_PATH gcc_include_path)
   if (WIN32)
     execute_process(COMMAND arm-none-eabi-gcc -v -x c -E NUL ERROR_VARIABLE _gcc_output OUTPUT_QUIET)
   else()
-    execute_process(COMMAND arm-none-eabi-gcc -v -x c -E - INPUT_FILE /dev/null ERROR_VARIABLE _gcc_output OUTPUT_QUIET)
+    execute_process(COMMAND arm-none-eabi-gcc argument -v -x c -E - INPUT_FILE /dev/null ERROR_VARIABLE _gcc_output OUTPUT_QUIET)
```

Comment:
I don't see what is this "argument"?

Reply:
if it was something for cmake, it should be ARGUMENT, but I don't see this in the documentation of execute_process:
https://cmake.org/cmake/help/v3.18/command/execute_process.html